cpp/src/gandiva/tests/test_util.cc

```diff
@@ -17,22 +17,20 @@
 
 #include "gandiva/tests/test_util.h"
 
-#include <filesystem>
+#include "arrow/util/io_util.h"
 
 namespace gandiva {
 std::shared_ptr<Configuration> TestConfiguration() {
-  auto builder = ConfigurationBuilder();
-  return builder.DefaultConfiguration();
+  return ConfigurationBuilder::DefaultConfiguration();
```

Comment:
Previously this `DefaultConfiguration` function is called by creating a new instance of `ConfigurationBuilder`, but I think this is not necessary since it is a static function of `ConfigurationBuilder`, so I change it as well.